repos/fdbt-site/src/pages/selectPassengerType.tsx

```diff
@@ -0,0 +1,125 @@
+import { getAndValidateNoc, getCsrfToken } from '../utils';
+import { getSessionAttribute } from '../utils/sessions';
+
+const title = 'Select Passenger Type - Create Fares Data Service';
+const description = 'Select Passenger Type selection page of the Create Fares Data Service';
+
+interface PassengerTypeProps {
+    errors: ErrorInfo[];
+    csrfToken: string;
+    savedGroups: FullGroupPassengerType[];
+    savedPassengerTypes: SinglePassengerType[];
+}
+
+const SelectPassengerType = ({
+    errors = [],
+    csrfToken,
+    savedGroups,
+    savedPassengerTypes,
+}: PassengerTypeProps): ReactElement => (
+    <TwoThirdsLayout title={title} description={description} errors={errors}>
+        <CsrfForm action="/api/selectPassengerType" method="post" csrfToken={csrfToken}>
+            <>
+                <ErrorSummary errors={errors} />
+                <div className={`govuk-form-group ${errors.length > 0 ? 'govuk-form-group--error' : ''}`}>
+                    <fieldset className="govuk-fieldset" aria-describedby="passenger-type-page-heading">
+                        <legend className="govuk-fieldset__legend govuk-fieldset__legend--l">
+                            <h1 className="govuk-fieldset__heading" id="passenger-type-page-heading">
+                                Select a passenger type
+                            </h1>
+                        </legend>
+                        <div className="govuk-warning-text">
+                            <span className="govuk-warning-text__icon govuk-!-margin-top-1" aria-hidden="true">
+                                !
+                            </span>
+                            <strong className="govuk-warning-text__text">
+                                <span className="govuk-warning-text__assistive">Warning</span>
+                                You can create new types in your{' '}
+                                <a className="govuk-link" href="/viewPassengerTypes">
+                                    operator settings.
+                                </a>{' '}
+                                <br />
+                                Don&apos;t worry you can navigate back to this page when you are finished.
+                            </strong>
+                        </div>
+
+                        {savedPassengerTypes.length === 0 ? (
+                            <>
+                                <span className="govuk-body">
+                                    <i>You currently have no saved passenger types</i>
+                                </span>
+                            </>
+                        ) : (
+                            <div className="govuk-heading-m">
+                                <div className="govuk-radios" id="individual-passengers">
+                                    <h3>Individuals</h3>
+                                    <FormElementWrapper errors={errors} errorId={''} errorClass="govuk-radios--error">
```

Comment:
Basically we don't know the error id of one of the elements because it could be anything (depending on what they name it) - which is why we pass an empty string. It still renders the big red bar etc, so it still serves 90% of the functionality - the only thing we lose is the 'red outline' around the thing they did wrong (which is less important as the only way to misuse the page is not entering anything, which means theres no one specifically wrong element).

Reply:
I thought the big red bar was the FormGroupWrapper ?